tests/queries/0_stateless/01161_all_system_tables.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+# Tags: no-parallel
```

Comment:
It failed because `system.remote_data_paths` traverses all disks files, which could be deleted in parallel to this test.